python/nav/ipdevpoll/snmp/pynetsnmp.py

```diff
@@ -29,7 +29,6 @@
 os.environ['MIBDIRS'] = ''
 
 from pynetsnmp import twistedsnmp, netsnmp
-from pynetsnmp.twistedsnmp import snmpprotocol
```

Comment:
> not
> Please have a closer look if `pynetsnmp.twistedsnmp.snmpprotocol` is actually not needed.

As you can see from the test suite, this is definitely needed.

This is a replaceable shim module, its symbols are imported from several other modules, including third party symbols.

Perhaps declaring a module `__all__` value that includes `snmpprotocol` would prevent the linter from complaining?